lms/static/scripts/frontend_apps/api-types.ts

```diff
@@ -83,3 +83,28 @@ export type JSTORMetadata = {
 export type JSTORThumbnail = {
   image: string;
 };
+
+/**
+ * Configuration for an assignment.
+ */
+export type Assignment = {
+  id: string;
+  group_set_id: string;
+  document: {
+    url: string;
+  };
+};
+
+/**
+ * Response for an `/api/assignment/{id}/config` call.
+ *
+ * This includes the current assignment configuration and, currently,
+ * the metadata needed to launch the file picker app to reconfigure the
+ * assignment.
+ *
+ * TODO: Split file picker config into separate API call?
+ */
+export type AssignmentConfig = {
+  assignment: Assignment;
+  filePicker: Omit<FilePickerConfig, 'formAction' | 'formFields'>;
```

Comment:
I think we discussed using snake_case for config. Should `filePicker` be `file_picker` then?

If this references an already existing config, then feel free to ignore this comment.

Reply:
These types currently just document the config as added by Marcos in https://github.com/hypothesis/lms/pull/5060. We should indeed change these to `snake_case` in a future iteration though.

Reply:
Got it.